app/Http/Controllers/SupplyController.php

```diff
@@ -215,13 +215,16 @@ public function logNetworkView(Request $request, Adselect $adselect, $id)
 
         // GET kewords from aduser
         $impressionId = $request->query->get('iid');
-        $aduser_endpoint = config('app.aduser_endpoint');
+//        $aduser_endpoint = config('app.aduser_endpoint');
+        $aduser_endpoint = config('app.aduser_local_endpoint');
 
         if (empty($aduser_endpoint) || empty($impressionId)) {
             $log->save();
         // TODO: process?
         } else {
-            $userdata = json_decode(file_get_contents("{$aduser_endpoint}/getData/{$impressionId}"), true);
+//            $userdata = json_decode(file_get_contents("{$aduser_endpoint}/getData/{$impressionId}"), true);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)